scripts/communityModules/VRBuild/VRBuildGrabscale.js

```diff
@@ -0,0 +1,484 @@
+//
+//  VRBuildGrabScale.js
+//  
+//  Created by Basinsky on 2/2/2020
+//  
+//  Script to scale in seperate XYZ directions by grabbing 
+//
+//  Distributed under the Apache License, Version 2.0.
+//  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+//
+var isGrabLeftInProgres = false;
+var isGrabRightInProgres = false;
+var isGrabLeftAndRight = false;
+var grabbedLeftEntityID;
+var grabbedRightEntityID;
+var channelName = "Hifi-Object-Manipulation";
+var closeEntities = [];
+var entityToBeGrabbedID = null;
+var RIGHT_JOINT = MyAvatar.getJointIndex("RightHandMiddle1");
+var LEFT_JOINT = MyAvatar.getJointIndex("LeftHandMiddle1");
+var startHandDistance = 0;
+var isScaling = false;
+var newScale = 1;
+var oldDimensions;
+var newDimensions = {x: 1,y: 1,z: 1};
```

Comment:
Spaces missing inside braces and after commas: `{x: 1,y: 1,z: 1}` ==> `{ x: 1, y: 1, z: 1 }
Here and elsewhere.